fili-core/src/test/groovy/com/yahoo/bard/webservice/druid/model/query/GroupByQuerySpec.groovy

```diff
@@ -341,35 +355,39 @@ class GroupByQuerySpec extends Specification {
         String druidQuery2 = MAPPER.writeValueAsString(dq2)
         String druidQuery3 = MAPPER.writeValueAsString(dq3)
 
-        String postAgg1 = ("""[
-                            {
-                                "fieldName":"pageViewsSum",
-                                "type":"fieldAccess"
-                            }
-                        ]""").replaceAll(/\s/, "")
-
-        String postAgg2 = ("""[
+        String postAgg1 = """
+                [
+                    {
+                        "fieldName":"pageViewsSum",
+                        "type":"fieldAccess"
+                    }
+                ]
+        """
+
+        String postAgg2 = """
+                [
+                    {
+                        "fieldName":"pageViewsSum",
+                        "type":"fieldAccess"
+                    },
+                    {
+                        "name":"postAggDiv",
+                        "fields":
+                            [
                                 {
                                     "fieldName":"pageViewsSum",
                                     "type":"fieldAccess"
                                 },
                                 {
-                                    "name":"postAggDiv",
-                                    "fields":
-                                        [
-                                            {
-                                                "fieldName":"pageViewsSum",
-                                                "type":"fieldAccess"
-                                            },
-                                            {
-                                                "fieldName":"timeSpentSum",
-                                                "type":"fieldAccess"
-                                            }
-                                        ],
-                                    "type":"arithmetic",
-                                    "fn":"/"
+                                    "fieldName":"timeSpentSum",
+                                    "type":"fieldAccess"
```

Comment:
Hmm... I din't intend to delete anything. I just re-indent string to make it look more standardized.